napari/_qt/qt_main_window.py

```diff
@@ -137,9 +137,14 @@ def _add_file_menu(self):
         )
         open_folder.triggered.connect(self.qt_viewer._open_folder)
 
+        screenshot = QAction('Screenshot', self._qt_window)
+        screenshot.setStatusTip('Take screenshot')
```

Comment:
Do you want to add a shortcut too, maybe `screenshot.setShortcut("Ctrl+Alt+S")`?

Reply:
> Looks great! I'd be ok with adding Ctrl-Alt-S for the screenshot, but agree that it might be contentious. =)

lol - I just read this, I guess we're aligned ;-)